js/ad_manager_vast.js

```diff
@@ -697,17 +851,27 @@ OO.Ads.manager(function(_, $) {
           indexInPod: adWrapper.ad.adPodIndex,
           skippable: false
         });
-      }
-      else {
+      } else {
         var streamUrl;
         if (adWrapper.ad && adWrapper.ad.streamUrl) {
           streamUrl = adWrapper.ad.streamUrl;
         }
         else if (adWrapper.streamUrl) {
           streamUrl = adWrapper.streamUrl;
+        } else {
+          streamUrl = '';
         }
+
         this.amc.sendURLToLoadAndPlayNonLinearAd(adWrapper, adWrapper.id, streamUrl);
-        this.checkCompanionAds(adWrapper.ad);
+      }
+      this.checkCompanionAds(adWrapper.ad);
+      if (isVPaid) {
+        //Since IMA handles its own UI, we want the video player to hide its UI elements
+        this.amc.hidePlayerUi();
```

Comment:
There is an extra call to hidePlayerUI down in the _beginAd function. Can you check if one of these can be removed without affecting anything? Thanks!